bin/checkurls.ps1

```diff
@@ -1,4 +1,4 @@
 if(!$env:SCOOP_HOME) { $env:SCOOP_HOME = resolve-path (split-path (split-path (scoop which scoop))) }
 $checkurls = "$env:SCOOP_HOME/bin/checkurls.ps1"
 $dir = "$psscriptroot/../bucket" # checks the parent dir
-Invoke-Expression -command "$checkurls -dir $dir $($args | ForEach-Object { "$_ " })"
+Invoke-Expression -command "&'$checkurls' -dir '$dir' $($args | ForEach-Object { "$_ " })"
```

Comment:
Need a space after the &, per @Ash258

Reply:
Also you can easily skip Invoke expression. Since you are using call operator you do not need iex.

```diff
-Invoke-Expression -command "&'$checkurls' -dir '$dir' $($args | ForEach-Object { "$_ " })"
+& "$checkurls" -dir $dir $args # For each is not needed too I think. But not 100% sure
```